api/src/main/java/com/sevenbridges/apiclient/task/ExecutionStatus.java

```diff
@@ -76,4 +76,52 @@
    */
   Integer getAborted();
 
+  /**
+   * Boolean indicating whether the platform instance limit has bean reached.
+   *
+   * @return Boolean system limit
+   */
+  Boolean getSystemLimit();
+
+  /**
+   * Boolean indicating whether the account instance limit has bean reached.
+   *
+   * @return Boolean account limit
+   */
+  Boolean getAccountLimit();
+
+  /**
+   * Boolean indicating whether the task is in initializing state.
+   *
+   * @return Boolean instance init
+   */
+  Boolean getInstanceInit();
+
+  /**
+   * Long amount of time that task has spent in the Queued state.
+   *
+   * @return Long queued duration
+   */
+  Integer getQueuedDuration();
+
+  /**
+   * Long amount of time that task has spent in the Running state.
+   *
+   * @return Long running duration
+   */
+  Integer getRunningDuration();
+
+  /**
+   * Long amount of current execution time.
+   *
+   * @return Long current execution duration.
+   */
+  Integer getExecutionDuration();
+
+  /**
+   * Long total duration.
+   *
+   * @return Long total task duration.
+   */
+  Integer getDuration();
```

Comment:
Should have suffix describing the unit, in plural form, as well as suffix "total" as it represents accumulating count, ie. `getDurationSecondsTotal()` (maybe you would prefer `getTotalDurationSeconds()`, which was my first version).
This is important, as we will probably never change this.